pkg/daemon/daemon.go

```diff
@@ -767,24 +768,43 @@ func (dn *Daemon) syncNodeHypershift(key string) error {
 
 	glog.Infof("Update is reconcilable. Diff: %v", mcDiff)
 
-	// If we want to match existing MCD behaviour, we will also need to
-	// check whether we need to drain.
-	// TODO add disruptionless updates here
+	// This should be eventually de-duplicated with the update() function.
```

Comment:
I'm fine with not de-duplicating this as part of this PR, but we should open a bug to ensure that it gets addressed in the future.